deploy/spinnaker/kustomize/spinnakerservice.yml

```diff
@@ -7,11 +7,11 @@ spec:
   spinnakerConfig:
     # spec.spinnakerConfig.config - This section contains the contents of a deployment found in a halconfig .deploymentConfigurations[0]
     config:
-      version: 1.15.1   # the version of Spinnaker to be deployed
+      version: 1.17.1   # the version of Spinnaker to be deployed
       persistentStorage:
         persistentStoreType: s3
         s3:
-          bucket: mybucket
+          bucket: change-me # Change to a unique name. Spinnaker stores application and pipeline definitions here
```

Comment:
The Docs team is moving to angle brackets for [placeholders](https://kubernetes.io/docs/contribute/style/style-guide/#use-angle-brackets-for-placeholders).  So how about `<change-me>`... or `<globally-unique-bucket-name>`